### ISSUE TITLE
Permissions needed on Windows x64?

### ISSUE THREAD

**Issue by gordonfpanam** [mockingbirdnest/Principia#1341]
I read over this:

https://github.com/mockingbirdnest/Principia/wiki/Installing,-reporting-bugs,-and-frequently-asked-questions

...and read this:

> Make sure the permissions are set correctly on the folder containing KSP. If in doubt, set its permissions for that folder to Full Control for Everyone.

Not only is this poor security practice (general complaint about KSP, not specific to Principia), is isn't even necessary unless an application is directly manipulating file system permissions. "Modify" permission for the BUILTIN\Users local group should be enough, even for admin users operating with User Account Control enabled.

After verifying this behaviour with the current Principia release, would anyone mind if I edited that Wiki page accordingly?

**Comment by gordonfpanam** [comment]
Are there specific error entries I should be looking for in the glog folder or ksp.log that would indicate a permission or access-denied problem? Cardano appears to run on a Steam-based install of KSP 1.2.2, but the logs I'm reading so far don't indicate any permission problems. I also run as a non-admin user with UAC, SRP, DEP, and a bunch of other security TLAs included with Windows 10 Pro, and KSP tends to run without incident in this environment.

**Comment by eggrobin** [comment]
If it works (you have the Principia UI and it is doing its thing), it works; we wrote that because in some circumstances Principia would fail to load because of permission reasons (e.g. if KSP, not installed via Steam, was manually dropped in Program Files). The permission suggestions are clearly overly broad, but it's a sledgehammer that worked with our confused users.

**Comment by gordonfpanam** [comment]
This is a bigger problem with Kerbal Space Program than with any add-ons written for it.

But I do this sort of thing for a living. What about a simple script to address permissions that a non-technical user can just launch? Even this command line would do the job:

`icacls "%programfiles%\[ksp-folder]" /grant Users:(OI)(CI)M`

Sledgehammers tend to make holes that become difficult to repair; this coming from a big fan of Canada's Worst Handyman. If someone knows how to grant Everyone Full Control, they know how to do this.

**Comment by pleroy** [comment]
Thanks for noticing this.  I agree that it's a bad idea to teach users to circumvent proper permission control.

I wrote this blurb long ago, when KSP didn't come with a proper Windows installer, and I was annoyed that people who would uncompress the `zip` in `Program Files` or `Program Files (x86)` would run into mysterious permission problems.

But these days there is a proper installer, and I just verified that if I use it to install 1.2.2 and drop Principia at the right place it all works without any need to patch the permissions.  So I am removing this bit from the Wiki page.
